src/adaptation/Adaptation.jl

```diff
@@ -61,15 +53,40 @@ end
 initialize!(adaptor::NaiveHMCAdaptor, n_adapts::Int) = nothing
 finalize!(aca::NaiveHMCAdaptor) = finalize!(aca.ssa)
 
-##
-## Stan's windowed adaptor.
-##
-include("stan_adaption.jl")
+include("stan_adaptor.jl")
+export NaiveHMCAdaptor, StanHMCAdaptor
+
+# User helpers for AdvancedHMC
+
+using ..AdvancedHMC: 
+    AbstractIntegrator, nom_step_size,
+    AbstractMetric, UnitEuclideanMetric, DiagEuclideanMetric, DenseEuclideanMetric
+
+StepSizeAdaptor(δ::AbstractFloat, i::AbstractIntegrator) = 
+    NesterovDualAveraging(δ, nom_step_size(i))
+
+MassMatrixAdaptor(m::UnitEuclideanMetric{T}) where {T} = 
+    UnitMassMatrix{T}()
+MassMatrixAdaptor(m::DiagEuclideanMetric{T}) where {T} = 
+    WelfordVar{T}(size(m); var=copy(m.M⁻¹))
+MassMatrixAdaptor(m::DenseEuclideanMetric{T}) where {T} = 
+    WelfordCov{T}(size(m); cov=copy(m.M⁻¹))
+
+MassMatrixAdaptor(
+    m::Type{TM}, 
+    sz::Tuple{Vararg{Int}}=(2,)
+) where {TM<:AbstractMetric} = MassMatrixAdaptor(Float64, m, sz)
+
+MassMatrixAdaptor(
+    ::Type{T}, 
+    ::Type{TM}, 
+    sz::Tuple{Vararg{Int}}=(2,)
+) where {T, TM<:AbstractMetric} = MassMatrixAdaptor(TM(T, sz))
+
+# Deprecations
 
-export adapt!, initialize!, finalize!, getϵ, getM⁻¹, reset!, renew,
-       NesterovDualAveraging,
-       UnitPreconditioner, DiagPreconditioner, DensePreconditioner,
-       AbstractMetric, UnitEuclideanMetric, DiagEuclideanMetric, DenseEuclideanMetric,
-       Preconditioner, NaiveHMCAdaptor, StanHMCAdaptor
+@deprecate StanHMCAdaptor(n_adapts, pc, ssa) initialize!(StanHMCAdaptor(pc, ssa), n_adapts)
+@deprecate NesterovDualAveraging(δ, i::AbstractIntegrator) StepSizeAdaptor(δ, i)
```

Comment:
Perhaps move all the deprecation statements into a separate file.

Reply:
Can I ask why? I think the current way is fine. Functions are still located in whatever file they are in based on their functionatliy, and I can easily find all deprecated ones by searching for the keyword `@deprecate`.

Reply:
The search results is meaningful in the sense that it tells me what are the deprecated functions actually pointing to.

Reply:
It's a minor comment. The purpose is to track all depreciated APIs from a central location, and regularly remove them when a new major/breaking release is to be released.

Reply:
As I said, it's quick easy to track all the deprecated functions by searching `@deprecate`, which is quite simple in both terminal and text editors. So I guess we don't really need it.